client/client_test.go

```
@@ -1665,3 +1828,45 @@ func tmpdir(appliers ...fstest.Applier) (string, error) {
 	}
 	return tmpdir, nil
 }
+
+func makeSSHAgentSock(agent agent.Agent) (p string, cleanup func() error, err error) {
+	tmpDir, err := ioutil.TempDir("", "buildkit")
+	if err != nil {
+		return "", nil, err
+	}
+	defer func() {
+		if err != nil {
+			os.RemoveAll(tmpDir)
+		}
+	}()
+
+	sockPath := filepath.Join(tmpDir, "ssh_auth_sock")
+
+	l, err := net.Listen("unix", sockPath)
+	if err != nil {
+		return "", nil, err
+	}
+
+	s := &server{l: l}
+	go s.run(agent)
```

the error is ignored